e2e/etcdctlv3_test.go

```diff
@@ -230,6 +262,70 @@ func getTest(cx ctlCtx) {
 		}
 	}
 }
+
+func txnTestSuccess(cx ctlCtx) {
+	defer close(cx.errc)
+
+	if err := ctlV3Put(cx, "key1", "value1"); err != nil {
+		if cx.dialTimeout > 0 && isGRPCTimedout(err) {
```

Comment:
I don't think it makes sense to check if timeouts are permitted here and below. The rest of the test wouldn't work if it timed out...